cmake/Modules/FindLibAJANTV2.cmake

```diff
@@ -106,6 +106,17 @@ if(LIBAJANTV2_FOUND)
       ${AJA_LIBRARIES_INCLUDE_DIR}/ajabase ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2
       ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/includes)
 
+  if(WIN32)
+    set(AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIRS}
+                                   ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/win)
+  elseif(APPLE)
+    set(AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIRS}
+                                   ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/mac)
+  elseif(UNIX AND NOT APPLE)
+    set(AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIRS}
+                                   ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/lin)
```

Comment:
Please replace:
- `WIN32` by `OS_WINDOWS`
- `APPLE` by `OS_MACOS`
- `UNIX AND NOT APPLE` by `OS_LINUX` if it only targets Linux and not FreeBSD and OpenBSD

This is explained in OBS CMake files [here](https://github.com/obsproject/obs-studio/blob/master/plugins/CMakeLists.txt#L23)